### PR TITLE
Adding onLoaded callback for RedocStandalone

### PR DESCRIPTION
**Pull request by dlcognite**

### CODE REVIEW

File: packages/docusaurus-theme-redoc/src/types/common.ts [hunk 1]
```diff
@@ -5,6 +5,7 @@ import type { RecursivePartial } from './util';
 export type RedocProps = {
   spec?: Record<string, unknown>;
   specUrl?: string;
+  onLoaded: (e?: Error) => void;
 };
```
Comment (rohit-gohri) [inline packages/docusaurus-theme-redoc/src/types/common.ts:9]: @dlcognite What do you think about just extending the type to include everything and then passing the props as `...rest`?

Something like:

```suggestion
export interface RedocProps extends Omit<RedocStandaloneProps, 'spec' | 'specUrl' | 'options'> {
  spec?: Record<string, unknown>;
  specUrl?: string;
};
```

**Comment by dlcognite** [comment]
Awesome suggestion :)

File: packages/docusaurus-theme-redoc/src/types/common.ts [hunk 2]
```diff
@@ -2,7 +2,7 @@ import type { Props as LayoutProps } from '@theme/Layout';
 import type { RedocRawOptions, ObjectDescriptionProps } from 'redoc';
 import type { RecursivePartial } from './util';
 
-export type RedocProps = {
+export interface RedocProps extends Omit<RedocStandaloneProps, 'spec' | 'specUrl' | 'options'> {
```
Comment (rohit-gohri) [inline packages/docusaurus-theme-redoc/src/types/common.ts:5]: you forgot to import `RedocStandaloneProps`

**Comment by dlcognite** [comment]
Sorry was sick over the weekend, finally fixed :) great catch

**Pull request merged**